From 6fb8014c2474e086a66d1474bd771d256e79be8d Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Wed, 19 Sep 2012 20:59:36 +0000
Subject: Remove gsd_typing_break_plugin_class_finalize boilerplate

class_finalize is now included in the GNOME_SETTINGS_PLUGIN_REGISTER
convenience macro. Also bumped g-s-d requirement to 3.5.91 to get the
new macro.
---
diff --git a/configure.in b/configure.in
index d0adc9d..aaa1223 100644
--- a/configure.in
+++ b/configure.in
@@ -48,7 +48,7 @@ GDK_PIXBUF_REQUIRED=2.25.3
 GTK_REQUIRED=3.0.0
 LIBCANBERRA_REQUIRED=0
 LIBNOTIFY_REQUIRED=0.7
-GSD_REQUIRED=2.91.8
+GSD_REQUIRED=3.5.91
 
 # ************
 # Typing Break
diff --git a/src/gsd-typing-break-plugin.c b/src/gsd-typing-break-plugin.c
index ceb5e88..8150331 100644
--- a/src/gsd-typing-break-plugin.c
+++ b/src/gsd-typing-break-plugin.c
@@ -35,11 +35,6 @@ struct GsdTypingBreakPluginPrivate {
 GNOME_SETTINGS_PLUGIN_REGISTER (GsdTypingBreakPlugin, gsd_typing_break_plugin)
 
 static void
-gsd_typing_break_plugin_class_finalize (GsdTypingBreakPluginClass *klass)
-{
-}
-
-static void
 gsd_typing_break_plugin_init (GsdTypingBreakPlugin *plugin)
 {
         plugin->priv = GSD_TYPING_BREAK_PLUGIN_GET_PRIVATE (plugin);
--
cgit v0.9.2
